docket.mk

```diff
@@ -11,8 +11,12 @@ PHONY: sfm_pc/management/commands/country_data
 			exit 255 \
 		)'
 
+download_to_s3 : sfm_download.zip
+	aws s3 cp $< s3://sfm-public-data/
+
 sfm_download.zip : sfm_pc/management/commands/country_data
-	zip -r $@ $<
+	# move into the target directory, zip to the root dir, move back to root dir
+	cd $< && zip -r ../../../../$@ . && cd ../../../../
```

Comment:
Make commands run in their own sub-shells, so you don't need to switch directories at the end of the command. Also, can you not do something like `zip -r $@ $<` so you don't have to switch in the first place?